pkg/cbox/publicshare/sql/sql.go

```diff
@@ -348,6 +353,9 @@ func (m *manager) ListPublicShares(ctx context.Context, u *user.User, filters []
 		if expired(cs3Share) {
 			_ = m.cleanupExpiredShares()
 		} else {
+			if cs3Share.PasswordProtected && sign {
```

Comment:
This returns the password, not the sign

Reply:
What do you mean? The `cs3Share.PasswordProtected`? It's a boolean which is true when a password is set for the share.

https://github.com/cs3org/reva/blob/538c8bdafb0ceeca96f650e580885dfeb4035276/pkg/cbox/utils/conversions.go#L213-L215
https://github.com/cs3org/reva/blob/538c8bdafb0ceeca96f650e580885dfeb4035276/pkg/cbox/utils/conversions.go#L236

Reply:
Never mind